js/ui/form/components/label.js

```diff
@@ -37,6 +37,9 @@ export function renderLabel({ text, id, location, alignment, labelID = null, mar
         labelTemplate.render({
             container: getPublicElement($labelContent),
             model: labelTemplateData,
+            onRendered() {
+                onLabelsRendered?.();
```

Comment:
Are u sure that this `onRendered` will be executed the latest? What if we have 2 items in the dataSource, and labelTemplate for the first one renders later then the second one.

Reply:
https://github.com/DevExpress/DevExtreme/pull/22961/commits/1f68deb5b4165de0274328699c1969b626b6a1af